README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
+| Triggers (imagestream)|       |      |    |
+| Pruning of builds     |       |      |    |
+| Cancelling of builds  |       |      |    |
+| Chaining of builds    |       |      |    |
 | Image Caching         |       |      |    |
+| Max duration / TO     |       |      |    |
+| Parallel vs. serial execution   |       |      |    |
 | ImageStreams support  |       |      |    |
 | Entitlements          |       |      |    |
+| Global Proxy support  |       |      |    |
```

Comment:
@adambkaplan another story for us ... either something like `obu` grabs the CAs. and injects them as part of the tekton task prior to calling the image builder binary, or we consult buildv2 on updating their controller to create config maps that get auto injected

Reply:
https://github.com/redhat-developer/build/issues/57